libpsautohint/autohintexe.c

```diff
@@ -464,24 +447,8 @@ main(int argc, char* argv[])
             outGlyphs[i] = malloc(outputSizes[i]);
         }
 
-        result = AutoColorString(inGlyphs[0], fontinfo, outGlyphs[0], &outputSizes[0],
+        result = AutoColorString(inGlyphs[0], fontinfo, &outGlyphs[0], &outputSizes[0],
                                  allowEdit, allowHintSub, roundCoords, debug);
-        if (result == AC_DestBuffOfloError) {
```

Comment:
I do think you should check result here before you assign another value to result just below.